### PR TITLE
chore(*): update modeljob operator helm

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Update modeljob operator helm

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @judgeeeeee 

**Release note**:

```release-note
Update modeljob operator helm
```

### CODE REVIEW

File: manifests/modeljob-operator/templates/preset-image-configmap.yaml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+  caffemodel-extract: '{{ .Values.docker.registry }}/{{ .Values.extraction.caffemodel-extract }}'
+  netdef-extract: '{{ .Values.docker.registry }}/{{ .Values.extraction.netdef-extract }}'
+  graphdef-extract: '{{ .Values.docker.registry }}/{{ .Values.extraction.graphdef-extract }}'
+  h5-extract: '{{ .Values.docker.registry }}/{{ .Values.extraction.h5-extract }}'
+  mxnetparams-extract: '{{ .Values.docker.registry }}/{{ .Values.extraction.mxnetparams-extract }}'
+  onnx-extract: '{{ .Values.docker.registry }}/{{ .Values.extraction.onnx-extract }}'
+  savedmodel-extract: '{{ .Values.docker.registry }}/{{ .Values.extraction.savedmodel-extract }}'
+  torchscript-extract: '{{ .Values.docker.registry }}/{{ .Values.extraction.torchscript-extract }}'
+  pmml-extract: '{{ .Values.docker.registry }}/{{ .Values.extraction.pmml-extract }}'
+  ormb-storage-initializer: "{{ .Values.docker.registry }}/{{ .Values.extraction.ormb-storage-initializer  }}"
```
Comment (judgeeeeee) [inline manifests/modeljob-operator/templates/preset-image-configmap.yaml:15]:
```suggestion
  ormb-storage-initializer: "{{ .Values.docker.registry }}/{{ .Values.mdoel.ormb-storage-initializer  }}"
```

**Review by FogDong — COMMENTED** [review]
/lgtm

**Pull request merged**